.github/workflows/build-test-release.yml

```diff
@@ -557,3 +557,67 @@ jobs:
           artifacts: packages/*
           omitBody: true
           replacesArtifacts: true
+
+  publish-controller-image-marketplace-ownerd-ecr:
```

Comment:
On both these jobs, is `ownerd` a typo? Should it be `owner` or `owned`?

Reply:
yes, should be owned

Reply:
fixed